types/react-calendar-timeline/react-calendar-timeline-tests.tsx

```diff
@@ -50,23 +50,23 @@ type TimelineGroupCustom = TimelineGroup<{ data: string }>;
 type TimelineItemCustom = TimelineItem<{ data: string }, Moment>;
 
 const groups2: TimelineGroupCustom[] = [
-    { id: 1, title: 'group 1', data: '1' },
-    { id: 2, title: 'group 2', data: '1' },
+    { id: '1', title: 'group 1', data: '1' },
+    { id: '2', title: 'group 2', data: '1' },
 ];
 
 const items2: TimelineItemCustom[] = [
-    { id: 1, group: 1, title: 'item 1', start_time: moment(), end_time: moment().add(1, 'hour'), data: '1' },
+    { id: '1', group: '1', title: 'item 1', start_time: moment(), end_time: moment().add(1, 'hour'), data: '1' },
     {
-        id: 2,
-        group: 2,
+        id: '2',
+        group: '2',
         title: 'item 2',
         start_time: moment().add(-0.5, 'hour'),
         end_time: moment().add(0.5, 'hour'),
         data: '1',
     },
     {
-        id: 3,
-        group: 1,
+        id: '3',
```

Comment:
Keep some of them to be `numbers`. To make sure test tests it's compatible with old `number[]`.

Reply:
There are other tests data sets (eg. line 31) that still use numbers, but thanks for the commend, I realized I forgot to add the offending prop to the test. Added it in this one and reverted to a mix of numbers and strings.